vars/buildImage.groovy

```diff
@@ -0,0 +1,74 @@
+// Builds a container image and returns its name.
+// Available parameters:
+//    dockerFile    string -- DockerFile used for the buildconfig
+//    nameSpace     string -- Namespace name to run the build
```

Comment:
Does it even make sense to support this as a parameter? I don't think we have the ACLs to create objects in any other namespace anyway.